packages/util-waiter/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@aws-sdk/util-waiter",
+  "version": "1.0.0-rc.8",
+  "description": "Shared utilities for client waiters for the AWS SDK",
+  "dependencies": {
+    "tslib": "^1.8.0",
+    "@aws-sdk/abort-controller": "^1.0.0-rc.7"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.4",
+    "jest": "^26.1.0",
+    "typescript": "~4.0.2"
```

The packages have moved to ts `~4.1.2` in https://github.com/aws/aws-sdk-js-v3/pull/1735
```suggestion
    "typescript": "~4.1.2"
```